daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionVersionSpec.scala

```diff
@@ -16,27 +16,43 @@ import scala.collection.immutable.HashMap
 class TransactionVersionSpec extends WordSpec with Matchers {
   import TransactionVersionSpec._
 
+  import VersionTimeline.maxVersion
+
+  // FIXME: https://github.com/digital-asset/daml/issues/5164
+  // Currently the engine uses `TransactionVersions.minOutputVersion` or latter.
+  // #5164 should provide a more granular way to control the versioning.
+  // Once #5164 is resolved, update those tests with different values for `minOutputVersion`.
+  val minOutputVersion = TransactionVersions.minOutputVersion
+
   "assignVersion" should {
     "prefer picking an older version" in {
```

Comment:
Please reword the test descriptions. `prefer picking an older version` doesn't make sense anymore.